rj_gameplay/README.md

```diff
@@ -11,32 +11,44 @@ Stable code for gameplay.
 
 **Implementation - `rj_gameplay/`**
 
-Frequently changing code of gameplay .
+Frequently changing code of gameplay.
 
 **Tests - `tests/stp`**
 
 Unit tests for stp/.
 
 ## STP Framework
 
-Our gameplay library adapts the [STP
+Our gameplay library adopts the name of the [STP
 framework](https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.61.1972&rep=rep1&type=pdf)
-proposed by CMDragons.
+proposed by CMDragons, but is quite different. All implementation files for STP are in `rj_gameplay/` as clarified above.
 
-**Skill - `rj_gameplay/skill/`**
+**Play - `rj_gameplay/play/`**
+
+Handles full team behavior. Gets potential roles from a list of >=6 tactics,
+then assigns 6 of them at any time.
 
-Atomic robot behavior which consists of a behavior tree and calls a ROS action.
-(Examples: Move, Kick)
+(Examples: `basic_defense.py` in folder above)
 
 **Tactic - `rj_gameplay/tactic/`**
 
-Handles complex single robot behavior. Think of as one role in a play.
-(Examples: Goalie Receiver Waller)
+Handles role costing for one or more robots, such that Plays can choose
+higher-level ideas for each robot. Think of as abstraction layer between single-robot
+Skills and whole-team Plays.
```

Comment:
In this tactic description, I feel like I'm learning what plays are more compared to the actual plays description.